index.html

```diff
@@ -998,16 +998,112 @@
 
 </div>
 
-## Safety Systems {#safety}
+## Safety, Abuse, and Fraud {#safety}
+
+There are some cases where privacy and safety are in tension. Many of these cases are
+short-term, only existing until we develop a system that achieves both privacy and
+safety. Long-term cases are rare, but still occasionally come up. We can distinguish
```

Comment:
Can agree that accountability, investigation and auditing are alternatives to data destruction for achieving privacy? If not then the "until we develop a system" language doesn't make sense to me.

Counterabuse attack detection requires long-term access to fairly broad data. Investigation also has data needs that are hard to constrain. As long as privacy is based on destruction of data, I don't see how these safety needs will be met.

Perhaps we need to move beyond the assumption that the only way to implement privacy is to destroy the data. There are many alternative approaches. By comparison, matches are dangerous, but we don't ban them, we use accountability, investigation and auditing mechanisms to make them safe.

Reply:
Data minimization is an important principle for privacy, but definitely not the only way to support privacy, online or otherwise. My understanding is that there are multiple techniques for supporting user safety as well, not exclusively long-term individual surveillance.

If the document includes text saying the only way to implement privacy is to destroy the data (I don't think that's there, but there are a lot of revisions and I may be behind), that would be a useful separate issue to open.

Reply:
The language here about "short term" and "long term" is inconsistent with my experience. Almost all successful online fraud and malware depends on privacy to be viable. The safety teams I work with spend a substantial part of their time on privacy compliance and on implementing work-arounds for privacy-induced regressions.

The basic constraints for safety are creativity and input data. When enforcement does not have the data to identify individuals with specificity, it leads to collateral damage; false positives or false negatives.

Reply:
This text isn't in the PR anymore.